cmd/openshiftCIAddBranch.go

```
@@ -148,11 +165,14 @@ func init() {
 	cmd.MarkFlagRequired("branch")
 	cmd.Flags().StringVar(&f.repoDir, "repo-dir", "", "Repo Dir")
 	cmd.MarkFlagRequired("repo-dir")
+	cmd.Flags().StringVar(&f.repoDir, "olm-type", "", "Repo Dir")
+	cmd.MarkFlagRequired("olm-type")
```

We are ignoring the error returned from the `MarkFlagRequired`. Should we add helpful error messages for these flags.

---

Looking at this a second time the `olm-type` needs a better description

---

Yes, well spotted. I've improved the definition of the param. It also needed to be updated to match the param as passed here -> https://gitlab.cee.redhat.com/integreatly-qe/ci-cd/-/blob/master/pipelines/rhmi-release.groovy#L134

---

Does it matter that it's also defined by the parent command? https://github.com/integr8ly/delorean/blob/master/cmd/root.go#L115

---

I'm also just noticing now that the command being used in the pipeline is `
`delorean release openshift-ci-release` whereas I've updated `delorean release add-branch` 
To me it looks like the code is completely duplicated and `release add-branch` is not referenced in the release pipeline. This is strange. I will need to push further changes.